src/sentry/utils/redis.py

```diff
@@ -53,27 +54,60 @@ def make_rb_cluster(*args, **kwargs):
     return _make_rb_cluster(*args, **kwargs)
 
 
+class _RBCluster(object):
+    def supports(self, config):
+        return not config.get('is_redis_cluster', False)
+
+    def factory(self, **config):
+        return _make_rb_cluster(**config)
+
+    def __str__(self):
+        return 'Redis Blaster Cluster'
+
+
+class _RedisCluster(object):
+    def supports(self, config):
+        return config.get('is_redis_cluster', False)
+
+    def factory(self, **config):
+        nodes = map(lambda n: n[1], config.get('hosts').items())
```

Comment:
> Are you saying in the `redis.clusters` configuration, it should just be formatted differently than how we're formatting the rb hosts?

Yeah. I think the rb configuration is the way it is because the previous clustering tool (nydus) required this format and I'm not sure there is a compelling reason to continue using the "map with monotonically increasing integer keys that starts at 0 without gaps" format when an array/list type will work just fine.